doc/server/tutorial.rst

```diff
@@ -234,16 +239,24 @@ For example:
 
         def update_aggregate(self, updated_sensor, reading, old_reading):
             if update_sensor is None:
-                # Instantiation, set a fresh value.
-                return Reading(time.time(), Sensor.Status.NOMINAL, 0)
+                # Instantiation, calculate total for sensors already in target.
+                ...
+                return Reading(...)
             if reading is None:
                 # The sensor is being removed from the set.
-                return Reading(updated_sensor.timestamp, Sensor.Status.Nominal, self.value - updated_sensor.value)
+                ...
+                return Reading(...)
             if old_reading is None:
                 # The sensor is being added to the set.
-                return Reading(updated_sensor.timestamp, Sensor.Status.Nominal, self.value + updated_sensor.value)
+                ...
+                return Reading(...)
             # Otherwise, it's just a change.
-            return Reading(updated_sensor.timestamp, Sensor.Status.Nominal, self.value - old_reading.value + reading.value)
+            ...
+            return Reading(...)
+
+         def filter_aggregate(self, sensor):
+            ...
+            return True
```

Comment:
I actually quite liked the previous version, since it showed how you could use the old reading to incrementally update a sensor.

Reply:
I moved it to the examples directory, though I guess the assumption that the interested user would look there isn't necessarily warranted.

Should I copy it back?

Reply:
I'd say copy it back. Particularly now that I've shortened it a bit, it should be fairly readable.